src/vs/workbench/parts/execution/electron-browser/terminal.ts

```diff
@@ -11,6 +11,8 @@ if (env.isLinux) {
 		defaultTerminalLinux = 'x-terminal-emulator';
 	} else if (process.env.DESKTOP_SESSION === 'gnome' || process.env.DESKTOP_SESSION === 'gnome-classic') {
 		defaultTerminalLinux = 'gnome-terminal';
+	} else if (process.env.DESKTOP_SESSION === 'kde-plasma') {
```

Comment:
So this `if` should be entered regardless of the specific distribution, as long as it's using KDE and not on Debian (due to the first case)? [This comment](http://unix.stackexchange.com/questions/116539/how-to-detect-the-desktop-environment-in-a-bash-script#comment181412_116569) I found concerns me a little, is there a more reliable environment variable if this can be the `"default"` on KDE Fedora?